package.json

```diff
@@ -1,130 +1,35 @@
-  },
-  "type": "module",
-  "main": "./index.js",
-  "bin": {
-    "launchpad": "./index.js"
-  },
+  "name": "launchpad",
+  "private": "true",
+  "version": "0.0.1",
+  "description": "monorepo for @bluecadet/launchpad and friends",
   "scripts": {
-    "start": "node ./index.js",
+    "changeset": "changeset",
+    "start": "npm run start -w @bluecadet/launchpad -- ",
     "monitor": "npm run start -w @bluecadet/launchpad-monitor --",
     "monitor-kill": "npm run pm2 kill -w @bluecadet/launchpad-monitor --",
     "scaffold": "npm run start -w @bluecadet/launchpad-scaffold --",
     "content": "npm run start -w @bluecadet/launchpad-content --",
     "download": "npm run content --",
-    "test": "cd test && node test.js --",
+    "test": "npm run test -w @bluecadet/launchpad -- ",
     "docs": "npx jsdoc ./packages/ --match *.js -r -d ./docs/"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/bluecadet/launchpad.git"
   },
-  "author": {
-    "name": "Benjamin Bojko",
-    "email": "npm@bbojko.de"
-  },
-  "maintainers": [
-    {
-      "name": "Benjamin Bojko",
-      "email": "npm@bbojko.de"
-    },
-    {
-      "name": "Pete Inge"
-    }
-  ],
-  "contributors": [
-  ],
+  "contributors": [],
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/bluecadet/launchpad/issues"
   },
   "homepage": "https://github.com/bluecadet/launchpad#readme",
-  "dependencies": {
-    "@bluecadet/launchpad-content": "file:./packages/content",
-    "@bluecadet/launchpad-core": "file:./packages/core",
-    "@bluecadet/launchpad-dashboard": "file:./packages/dashboard",
-    "@bluecadet/launchpad-monitor": "file:./packages/monitor",
-    "@bluecadet/launchpad-scaffold": "file:./packages/scaffold",
-    "@bluecadet/launchpad-utils": "file:./packages/utils",
-    "@portabletext/to-html": "^1.0.3",
-    "@sanity/block-content-to-markdown": "^0.0.5",
-    "@sanity/client": "^3.3.2",
-    "@sindresorhus/slugify": "^2.1.0",
-    "airtable": "^0.11.1",
-    "async": "^3.2.3",
-    "auto-bind": "^5.0.1",
-    "axon": "^2.0.3",
-    "chalk": "^5.0.0",
-    "cli-progress": "^3.9.1",
-    "contentful": "^9.0.0",
-    "cross-spawn": "^7.0.3",
-    "formdata-node": "^4.3.1",
-    "fs-extra": "^10.0.0",
-    "get-urls": "^10.0.0",
-    "got": "^12.3.0",
-    "jsonpath": "^1.1.1",
-    "logform": "^2.3.2",
-    "markdown-it": "^12.2.0",
-    "moment": "^2.29.1",
-    "p-debounce": "^4.0.0",
-    "p-queue": "^7.1.0",
-    "pm2": "^5.1.2",
-    "rimraf": "^3.0.2",
-    "sanitize-html": "^2.5.1",
-    "semver": "^7.3.5",
-    "sharp": "^0.30.7",
-    "strip-json-comments": "^4.0.0",
-    "sudo-prompt": "^9.2.1",
-    "tail": "^2.2.4",
-    "winston": "^3.4.0",
-    "winston-daily-rotate-file": "^4.5.5",
-    "yargs": "^17.3.1"
-  },
   "workspaces": [
     "./packages/*"
   ],
-  "optionalDependencies": {
-    "ffi-napi": "^4.0.3",
-    "ref-napi": "^3.0.3",
-    "robotjs": "^0.6.0"
-  },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
@claytercek should these be `devDependencies` since these aren't needed when we install launchpad in production environments?

Reply:
Yeah I think it can go either way here, because any dependencies in this high-level `package.json` will only be used for development because this isn't actually a publishable package.

Reply:
How about if we wanted to install this as a GH dependency? We've done this in the past to work off development builds. Would that still point to this root `package.json` or do you know if NPM would be smart enough to pull `@bluecadet/launchpad` this repo?

Reply:
I'd have to do some testing to see if we could get it to work as a GH dependency, but my hunch is that it will break. However, changesets does give a couple of tools for distributing development builds:
- [prereleases](https://github.com/changesets/changesets/blob/main/docs/prereleases.md)
- [snapshot-releases](https://github.com/changesets/changesets/blob/main/docs/snapshot-releases.md)

Reply:
Lol "Warning! Prereleases are very complicate" -- I'll take a look 🙈